IPython/core/magics/pylab.py

```diff
@@ -42,6 +42,8 @@ class PylabMagics(Magics):
     @skip_doctest
     @line_magic
     @magic_arguments.magic_arguments()
+    @magic_arguments.argument('-l', '--list', default=False, action='store_true',
```

Comment:
You don't actually need the `default=False` here - when you use store_true, the default is automatically False.